packages/antd/src/templates/FieldTemplate/index.js

```diff
@@ -64,7 +64,7 @@ const FieldTemplate = ({
       ) : (
         <Form.Item
           colon={colon}
-          // extra={!!rawDescription && description}
+          extra={!!rawDescription && description}
```

Comment:
Do you know what the `!!rawDescription` is doing here?

Reply:
As I see  `description` is an instance of `DescriptionField`, and `rawDescription` is just a plain text. So here we check whether `rawDescription` is not falsy and pass a `DescriptionField` to extra prop of antd Form.Item.
BTW, `DescriptionField` returns just `<span id={id}>{description}</span>`  for antd so nothing changes visually also if we just pass a `rawDescription` to `extra`.

Reply:
Checked that and seems like it works without `rawDescription` well. Antd `form.item` handles null/undefined description without additional checks on our side.

Reply:
Thanks. Do you know what `rawDescription` is even used for then?? Is it ever used in other widgets / themes / elsewhere in the codebase?

Reply:
It is not used anywhere else in antd theme now. Possibly in some other themes you should pass a raw string, not react object to make it work. Anyway it looks like to be out of scope of this PR related to antd theme only.